spec.example.yaml

```diff
@@ -38,24 +38,44 @@ loadGenerator:
     - name: v1-5-2
     # Groups of queries that are launched independently.
     queryGroups:
-    - intervalSeconds: 4
+    - name: FirstGroup
+      intervalSeconds: 4
       queries:
-      - go_goroutines
-      - container_memory_rss
-      - kube_pod_container_info
-      - codelab_api_http_requests_in_progress
-      - codelab_api_requests_total
-    - intervalSeconds: 10
+      - type: query_range
+        queryString: go_goroutines
+        start: 1489165548
+        end: 1489165569
```

Comment:
Looking at it, we probably also want to make "instant" vs "range" as well as the range parameters for the entire group for simplicities sake.
That should simulate things like dashboards very well where multiple widgets fire queries for the same time range at the same resolution